File: benchmark/php/task8_http.php
<?php
function fetch_url(string $url): array {
    $content = @file_get_contents($url);
    if ($content === false) {
        return ['success' => false, 'error' => 'Failed to fetch URL'];
    }
    return ['success' => true, 'body' => $content];
}

File: benchmark/perl/task8_http.pl
use LWP::Simple;

sub fetch_url {
    my ($url) = @_;
    my $content = get($url);
    if (defined $content) {
        return { success => 1, body => $content };
    }
    return { success => 0, error => "Failed to fetch URL" };
}
